lib/src/builder.dart

```diff
@@ -141,10 +141,10 @@ class AggregateTestBuilder extends Builder {
     if (imports.isEmpty) return;
     if (_randomSeed != null) {
       log.info(
-          'Shuffling test order with `randomize_ordering_seed: $_randomSeed`\n');
-      mains.shuffle(Random(_randomSeed));
+          'Shuffling test order with `randomize_ordering_seed: ${_randomSeed!}`\n');
```

Comment:
I'm surprised these `!` are needed with the `if (_randomSeed != null) {` check above. I think the analyzer should be smart enough to promote the var to non-nullable inside this block. Maybe that's something they fixed in newer Dart versions.

Reply:
hmm, I think these can be removed even in Dart 2.13.4

Reply:
It's because `_randomSeed` is a getter (that wraps `__randomSeed`), so there's no guarantee that the getter will return the same thing at each call site. I've worked around this with a local variable in the latest commit.